test/run-tests

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env emacs --script
+
+;; Adapted from Cider tests
+;; This script must be run from the test directory
+
+(let ((current-directory (file-name-directory load-file-name)))
+  (setq ess-test-path (expand-file-name "." current-directory))
+  (setq ess-root-path (expand-file-name "../lisp/" current-directory)))
+
+(add-to-list 'load-path ess-root-path)
+(add-to-list 'load-path ess-test-path)
+
+(require 'ess-site)
+(load (expand-file-name "ess-tests.el" ess-test-path) nil t)
+
+
+;;; (Re)generate files with test cases for indentation
+
+;; The RRR file is taken as a model, so modify this file to add or
+;; adjust test cases.
+(when (member "--generate-indent-cases" argv)
```

Comment:
How about a separate ./generate-indent-case file? It doesn't feel right to mix it with ./run-tests, unless you don't want to add other files in the git repo. But why wouldn't you? I think if all style files are in repo it can be used by people to compare styles and we can reference directly patterns in  those files.